test/kubemark/pre-existing/util.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function run-cmd-with-retries {
+  # Duplicate logic from run-gcloud-compute-with-retries
+  RETRIES="${RETRIES:-3}"
+  for attempt in $(seq 1 ${RETRIES}); do
+    exec 5>&1 # Duplicate &1 to &5 for use below.
+    result=$("$@" |& tee >(cat - >&5)) || local ret_val="$?"
```

Comment:
The '|&' changed to '2>&1' recently to make it work on older bash versions too. Sorry for asking to change this time and again.